### PR TITLE
Fix SHi and SSRF capability test

### PR DESCRIPTION
**Pull request by CarlesDD**
## Motivation

Assign the correct feature to SHi and SSRF capability test

## Changes

Fix test name and assigned feature

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] If PR title starts with `[<language>]`, double-check that only `<language>` is impacted by the change
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

### CODE REVIEW

File: tests/appsec/rasp/test_shi.py [hunk 1]
```diff
@@ -173,10 +173,10 @@ def test_ssrf_telemetry(self):
 
 
 @rfc("https://docs.google.com/document/d/1vmMqpl8STDk7rJnd3YBsa6O9hCls_XHHdsodD61zr_4/edit#heading=h.mshauo3jp6wh")
```
Comment (simon-id) [inline tests/appsec/rasp/test_shi.py:175]: is this the correct RFC ?

Comment (uurien) [reply tests/appsec/rasp/test_shi.py:175]: I don't think so, it should be the same that we have here: https://github.com/DataDog/system-tests/pull/3215/files#diff-605b24a0db1fc8609eaaf3f6521b33356179a5e4174a62bd5d071e7ec5dfa54eR82

```suggestion
@rfc("https://github.com/DataDog/system-tests/pull/3215/files#diff-605b24a0db1fc8609eaaf3f6521b33356179a5e4174a62bd5d071e7ec5dfa54eR82")
```

Comment (CarlesDD) [reply tests/appsec/rasp/test_shi.py:175]: Yes, it is `[RFC] Exploit prevention in the ASM libraries`, pointing to `Local & Remote Configuration` section where capabilities are described. What's the problem?

Comment (CarlesDD) [reply tests/appsec/rasp/test_shi.py:175]: Got it, didnt know there was an specific RFC for SHi

**Review by uurien — APPROVED** [review]

**Pull request merged**